Convert pogo-feedstock to v1 feedstock

beenje opened the pull request:
This PR converts pogo-feedstock to a v1 recipe and switch the conda build tool to rattler-build.
It has been automatically generated with [feedrattler v0.3.13](https://github.com/hadim/feedrattler).

Changes:
- [x] 📝 Converted `meta.yaml` to `recipe.yaml`
- [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
- [x] 🔧 Updated `conda-forge.yml` to use `rattler-build` and `pixi` (optional)
- [x] 🔢 Bumped the build number
- [x] 🐍 Applied temporary fixes for `python_min` and `python_version`
- [x] 🔄 Rerender the feedstock with conda-smithy
- [x] Ensured the license file is being packaged.

Bump the version to 9.10.5 at the same time

beenje committed:
Convert to v1 feedstock and use pixi as conda install tool

conda-forge-admin commented:
Hi! This is the friendly automated conda-forge-linting service.

I just wanted to let you know that I linted all conda-recipes in your PR (```recipe/recipe.yaml```) and found it was in an excellent condition.

beenje committed:
Bump version to 9.10.5

beenje committed:
Improve v1 recipe

beenje committed:
Update doc url

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.0 and conda-forge-pinning 2025.08.28.10.19.34

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.5

beenje committed:
Remove pixi as install tool

It's not available for ppc64le

beenje committed:
MNT: Re-rendered with conda-smithy 3.52.0 and conda-forge-pinning 2025.08.28.10.19.34

Other tools:
- conda-build 25.7.0
- rattler-build 0.46.0
- rattler-build-conda-compat 1.4.5

beenje committed:
Fix windows test

beenje commented:
@dlacoste-esrf I see you haven't run the deploy job yet in https://gitlab.com/tango-controls/pogo/-/pipelines/2007443552

I'll wait for your approval.
